_posts/2016-08-30-foreman-community-newsletter---august-2016.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Foreman Community Newsletter - August 2016
+date: 2016-08-30 21:59:12
+author: Greg Sutcliffe
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  Foreman 1.12.2 security fix, Discovery 6.0 released, Webpack support merged, tons
+  of plugin updates... it's been a busy month!
+---
+
+August is coming to a close, so it's time for your monthly dose of Foreman
+roundup. We've got releases, new core technology, YouTube videos and the usual
+long list of plugin updates.
+
+### 1.12.2 released
+
+The latest point release of the Foreman 1.12 series is now available. This is
+mainly a security release, containing two CVE fixes ([6319][6319] &
+[6320][6320]) as well as updating Rails to 4.2.7.1 which contains security
+fixes in Rails itself. As usual, more detail can be found in the
+[announcement][1_12_2]
+
+[6319]: https://theforeman.org/security.html#2016-6319
+[6320]: https://theforeman.org/security.html#2016-6320
+[1_12_2]: https://groups.google.com/d/topic/foreman-users/NXI3H30jTfI/discussion
+
+### Webpack for JavaScript assets
+
+One major change in the Foreman ecosystem is we're now using Webpack and NPM to
+build our assets. This allows us to move forward with newer assets that aren't
```

Comment:
It seemed an important enough change to mention it. I'm not sure if a dedicated Dev section would get enough usage, but I could add "Development Changes" to the section header (i.e. "## Development Changes - Webpack for JavaScript assets")